doc/enterprise-shield/p_enterprise_shield_config.md

```diff
@@ -0,0 +1,418 @@
+
+    -   Set [ssl.verify-client](../admin-reference/r_configure_gateway_service.md#sslverify-client) to require that the internal Gateway provide a digital certificate to establish an authorized connection.
+
+        ``` xml:
+        <ssl.verify-client>required</ssl.verify-client>
+        ```
+
+        For added security, you can use the `socks.ssl.verify-client` connect option to require the internal Gateway to provide a client digital certificate to establish a secure connection. This configuration establishes mutual authentication by ensuring that both the DMZ Gateway and internal Gateway are verified via TLS/SSL before transmitting data.
+
+    -   Configure [ws.inactivity.timeout](../admin-reference/r_configure_gateway_service.md#wsinactivitytimeout) to detect network failures between the DMZ and internal Gateways:
+    
+      ``` xml:
+        <ws.inactivity.timeout>60 seconds</ws.inactivity.timeout>
+      ```
+      Our example specifies that the network connection can be inactive for a maximum of 60 seconds. The internal Gateway drops the connection if communication with the DMZ Gateway is not possible for a period of 60 seconds.
+
+### Set up the App2 service on the DMZ Gateway
+
+The service configuration for the App2 service is exactly the same as for App1, except you need to provide a unique name for this service in the `name`, `accept`, and `connect` elements. The following example shows a complete service configuration for App2.
+
+``` xml:
+  <service>
+
+    <name>DMZ App2</name>
+    <accept>wss://gateway.example.com:8443/app2</accept>
+    <connect>wss://gateway.example.com:8443/app2</connect>
+
+    <type>proxy</type>
+
+    <connect-options>
+      <http.transport>socks://${dmz.backplane.hostname}:1080</http.transport>
+      <socks.mode>reverse</socks.mode>
+      <socks.timeout>2 seconds</socks.timeout>
+      <socks.transport>wsn+ssl://${dmz.backplane.hostname}:1080/shield</socks.transport>
+      <ssl.verify-client>required</ssl.verify-client>
+      <ws.inactivity.timeout>60 seconds</ws.inactivity.timeout>
+    </connect-options>
+
+    <cross-site-constraint>
+      <allow-origin>*</allow-origin>
+    </cross-site-constraint>
+
+  </service>
+```
+
+As you can see, the `connect-options` are exactly the same for both the App1 and App2 services.
+
+You've completed configuring the DMZ Gateway! Next we will configure security for the connection between the DMZ and internal Gateways.
+
+Step 6: Configure Security Between the DMZ and Internal Gateways
+------------------------------------------------------------------------------------------
+
+Each Gateway maintains two databases for SSL connections: a keystore that stores SSL certificates to present to a client, and a truststore that stores keys from trusted sources. The following example shows the security block in the Gateway configuration file:
```

Comment:
I'm not keen on the word "databases" here, it muddies the water, to me. How about "stores"?